### PR TITLE
Added postman collections, updated contexts and schemas and fixed links

### PR DESCRIPTION
**Pull request by umbreak**

### CODE REVIEW

File: src/main/paradox/docs/api/assets/project-postman.json [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
...kIiA6ICJONS1CU0ZxZG5NS3Y4SWtKUkg1R3E0LVA2c1RWQUxwU0EydGNQeEpWM1NBIn0.eyJqdGkiOiJmYjllNmY5NC0xM2FiLTQyNTQtYmI1MC02ZTg4ZGNhOTUxNmYiLCJleHAiOjE1MzkzMzAwNzUsIm5iZiI6MCwiaWF0IjoxNTM4NzQyMDY1LCJpc3MiOiJodHRwczovL2JicHRlYW0uZXBmbC5jaC9hdXRoL3JlYWxtcy9CQlAiLCJhdWQiOiJiYnAtbmV4dXMtcHJvZHVjdGlvbiIsInN1YiI6ImY6OWQ0NmRkZDYtMTM0ZS00NGQ2LWFhNzQtYmRmMDBmNDhkZmNlOmRtb250ZXJvIiwidHlwIjoiQmVhcmVyIiwiYXpwIjoiYmJwLW5leHVzLXByb2R1Y3Rpb24iLCJhdXRoX3RpbWUiOjE1Mzg3MjUyNzUsInNlc3Npb25fc3RhdGUiOiJlZmY1YWM0NS02YzY0LTRmYzUtOGI0YS1jYWU4YzFhMjliZTgiLCJhY3IiOiIwIiwiYWxsb3dlZC1vcmlnaW5zIjpbIi8qIl0sInJlc291cmNlX2FjY2VzcyI6e30sInNjb3BlIjoib3BlbmlkIG5leHVzIiwibmFtZSI6IkRpZGFjIE1vbnRlcm8gTW9udGVybyBNZW5kZXoiLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJkbW9udGVybyIsImdpdmVuX25hbWUiOiJEaWRhYyBNb250ZXJvIiwiZmFtaWx5X25hbWUiOiJNb250ZXJvIE1lbmRleiIsImVtYWlsIjoiZGlkYWMubW9udGVyb21lbmRlekBlcGZsLmNoIn0.qvX_IQzGNS-1oi8TrbMIpzRJURSy-Q7Deb8Lyv8UH66Cl6YIUOIDd5dF2EO9KpoEbZeCn7F2-W58R6TR0crQDlZxoRKdDwbfWAbpf4p0_G_p_qF9IOYs94IgC8rzU-uEP0gF83qzdlZ9qZva9fNSL8FYxKXeWPvWwvZCASu-HZL1PphXIFH3MqZMf4SaiMFGP-h_t0G5FBUqZVCThLifwrhz6GXGsZjGJHlu-M9kZNdTWWFtBe5-5URBunquAPnyV6VBwu2BL1umaqPMUW5tRumhi3OpMoww8JKQhzSsEVs5TcJF7zYlsVWFijKn7i_zYYgpELlt1JGmTc1VnByAJQ"
```
Comment (hygt) [inline src/main/paradox/docs/api/assets/project-postman.json]: You left one token.

File: src/main/paradox/docs/index.md [hunk 2]
```diff
@@ -53,20 +53,20 @@ This service is the heart of the BlueBrain Nexus platform. It allows users to de
 knowledge graph with data, attach files to data. It also provides semantic search facilities to discover similar and
 relevant data in the platform.
 
-[Source Code](https://github.com/BlueBrain/nexus-kg) | [Documentation](./kg)
+[Source Code](https://github.com/BlueBrain/nexus-kg) | @ref:[Documentation](./api/kg/index.md)
```
Comment (bogdanromanx) [inline src/main/paradox/docs/index.md]: why the `@ref`?

File: src/main/paradox/contexts/resolver/index.json [hunk 3]
```diff
@@ -1,6 +1,19 @@
 {
   "@context": {
     "nxv": "https://bluebrain.github.io/nexus/vocabulary/",
+    "nxs": "https://bluebrain.github.io/nexus/schemas/",
```
[the remark this answers is not in the document]
Comment (umbreak) [reply src/main/paradox/contexts/resolver/index.json]: I want to have them in a self contain json in this case.

File: src/main/paradox/docs/index.md (hunk 2 above)
Comment (umbreak) [reply src/main/paradox/docs/index.md]: [@ref](https://developer.lightbend.com/docs/paradox/current/directives/linking.html#ref-link) is going to verify that the link is correct when running `paradox` command, avoiding wrong links

Comment (bogdanromanx) [reply src/main/paradox/docs/index.md]: then please also add it to admin link

**Pull request merged**